quipucords/api/scantask/test_scantask_get_facts.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (C) 2022  Red Hat, Inc.
+
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+
+"""Test ScanTask.get_facts."""
+
+import json
+
+import pytest
+
+from api.models import (
+    JobInspectionResult,
+    RawFact,
+    ScanJob,
+    ScanTask,
+    SystemInspectionResult,
+    TaskInspectionResult,
+)
+from tests.utils.facts import random_name, random_value
+
+
+@pytest.fixture
+def raw_facts_list():
+    """Fixture representing a list of random facts."""
+    systems = []
+    for _ in range(50):
+        systems.append({random_name(): random_value() for _ in range(100)})
+    return systems
+
+
+@pytest.fixture
+def inspection_scantask(raw_facts_list):
+    """Scantask instance mapped to raw_facts_list."""
+    task_inspection_result = TaskInspectionResult.objects.create(
+        job_inspection_result=JobInspectionResult.objects.create()
+    )
+    for i, facts in enumerate(raw_facts_list):
```

Comment:
lint is complaining about the variable i, saying it doesn comply with the snake case rule. Maybe changing it to index would mute this warning?

Reply:
Mmm maybe this is a side effect from adding "db" to the list of "good-words"

Reply:
yeah.. here it is: https://github.com/PyCQA/pylint/blob/main/pylintrc#L234

Reply:
funny it isn't complaining about the `_` :eyes:

Reply:
Insted of changing the code I included the "good names" from the original pylintrc. what do u think, @nicolearagao ?

Reply:
good idea